common/networkpolicies/base/seldon.yaml

```diff
@@ -1,5 +1,5 @@
-kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
@juliusvonkohout Seldon isn't installed by default, so does it make sense to add this policy by default?

Reply:
That depends. We can also think about moving it to /contrib/seldon